buildSrc/src/main/resources/com/uber/okbuck/core/util/transform/BUCK_FILE

```diff
@@ -0,0 +1,37 @@
+import re
+
+file_name_regex = re.compile(r'^.*/([^/]+)\.(jar)$', re.IGNORECASE)
+
+jars = []
+configs = []
+
+for jar in glob(['*.jar']):
+  name = file_name_regex.sub(r'\1', jar)
+  jars.append(name)
+
+for config in glob(['*.config']):
```

Comment:
this is weird. there is no docs suggesting config files should have the extension. This is limiting. can we make it generic?